src/Assets/svg/CloseButton.tsx

```diff
@@ -0,0 +1,19 @@
+const CloseButton = () => {
+  return (
+    <svg
+      width="16"
+      height="17"
+      viewBox="0 0 16 17"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M15 1L8 8.5M1 16L8 8.5M8 8.5L1 1L15 16"
+        stroke="black"
+        strokeWidth="2"
+      />
+    </svg>
+  );
+};
+
+export default CloseButton;
```

Comment:
```suggestion
const CloseButton = () => (
  <svg
    width="16"
    height="17"
    viewBox="0 0 16 17"
    fill="none"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M15 1L8 8.5M1 16L8 8.5M8 8.5L1 1L15 16"
      stroke="black"
      strokeWidth="2"
    />
  </svg>
);

export default CloseButton;

```
이런 식으로 줄이는게 좋을 것 같아요

Reply:
넵! 감사합니다!